cellxgene_schema_cli/cellxgene_schema/convert.py

```diff
@@ -0,0 +1,104 @@
+import anndata as ad
+import numpy as np
+from . import ontology
+
+
+def convert(input_file, output_file):
+    print(f"converting {input_file} into {output_file}")
+    
+    dataset = ad.read_h5ad(input_file)
+
+    # Rename ethnicity_ontology_term_id field
+    dataset.obs.rename(columns={"ethnicity_ontology_term_id": "self_reported_ethnicity_ontology_term_id"}, inplace=True)
+    # For existing datasets with labels already added
+    dataset.obs.rename(columns={"ethnicity": "self_reported_ethnicity"}, inplace=True)
```

Comment:
helpful for reducing error output noise during mass automated conversion of datasets that already have added labels. Will have no effect on datasets without labels added.

Edge case: If someone erroneously adds an ethnicity column in addition to a self_reported_ethnicity column, could be confusing, but the docs already state that both self_reported_ethnicity and ethnicity are reserved names. So, validation should fail regardless.